src/bundle/Resources/public/scss/_labels.scss

```diff
@@ -0,0 +1,21 @@
+.ez-label {
+    margin-bottom: 0;
+    font-weight: 700;
+
+    &:after {
+        content: ': ';
+    }
+}
+
+.ez-label.checkbox-inline,
+.ez-label.form-check-label {
```

Comment:
It should go inside the `.ez-label {` declaration like this: `&.form-check-label`

Reply:
Ok.